lib/interfaces/src/VCFInterface.cpp

```diff
@@ -0,0 +1,25 @@
+#include "VCFInterface.h"
+#include "hytech.h"
+
+void VCFInterface::receive_pedals_message(const CAN_message_t& msg, unsigned long curr_millis)
+{
+    VCR_PEDALS_t vcr_pedals_msg;
```

Comment:
Is VCR_Pedals the CAN message? Why not use the existing pedals data message? Why label it VCR based on its destination?